Added Explode validator documentation

renanliberato opened the pull request:
I've wrote the basic documentation for Explode validator.

It came useful for me lately so I'd like to contribute with it.

It fixes both #171 and #137 , which are duplicates.

froschdesign commented on doc/book/validators/explode.md:
```diff
@@ -0,0 +1,30 @@
+
+The following options are supported for `Zend\Validator\Explode`:
+
+- `valueDelimiter`: Defines the delimiter used to explode the value to an array. It defaults to `,`. If working with an array, this option isn't used.
```
@renanliberato 
> If working with an array, this option isn't used.

We should clarify that the given "value" is meant here.

froschdesign commented on doc/book/validators/explode.md:
```diff
@@ -0,0 +1,30 @@
+]);
+
+$value  = [1, 4, 6, 8];
+$return = $valid->isValid($value);
```
`$valid` is wrong here.

I would change the example:

```php
$explodeValidator->isValid([1, 4, 6]);    // returns true
$explodeValidator->isValid([1, 4, 6, 8]); // returns false
```

froschdesign commented on doc/book/validators/explode.md:
```diff
@@ -0,0 +1,30 @@
+
+$value  = [1, 4, 6, 8];
+$return = $valid->isValid($value);
+// returns false
```
Can be deleted.

froschdesign commented on doc/book/validators/explode.md:
````diff
@@ -0,0 +1,30 @@
+// returns false
+```
+
+The above example returns `true` if all $value items are between 1 and 6.
````
Can be deleted, because is covered now in the example.

froschdesign replied:
Please add another example which demonstrate the usage of `valueDelimiter`. Thanks!

renanliberato commented:
Hi @froschdesign ,

I added the valueDelimiter example and improved the valueDelimiter description.

I also appreciated your suggestions and incorporated them into the chapter.

weierophinney commented:
Thanks, @renanlibrato!
